src/utils/element-utils.ts

```diff
@@ -1,8 +1,11 @@
 import { TOOLBAR_CONTAINER_CLASS, TOOLBAR_ID } from '../constants'
 
-export function isElementInToolbar(el: Element): boolean {
-    // NOTE: .closest is not supported in IE11 hence the operator check
-    return el.id === TOOLBAR_ID || !!el.closest?.('.' + TOOLBAR_CONTAINER_CLASS)
+export function isElementInToolbar(el: EventTarget | null): boolean {
+    if (el instanceof Element) {
```

Comment:
Needs an if here now, because `EventTarget` does not have `id` and `closest`.

Reply:
That doesn't work type wise. `isElementNode` only accepts Element as a param, but el is EventTarget. So I don't see an alternative to the instanceof right now

Reply:
Ah, of course... cool cool and good general support for instanceof so pretty safe 🤞